community/contributing/index.md

```diff
@@ -22,3 +22,8 @@ In the meantime, if you're really excited about contributing and don't want to w
 
 For more details about the patch submission process, look at [How to submit a patch](/community/contributing/submitting/).
 
+## How to run Spinnaker Locally
```

Comment:
I would call this "How to run Spinnaker locally for development", to avoid confusing users who just want to run Spinnaker, not develop against it

Reply:
Done.